cmd/tfplugindocs/testdata/scripts/schema-json/migrate/time_provider_success.txtar

```diff
@@ -0,0 +1,909 @@
+-- exp-examples/resources/rotating/import_2.sh --
+$ terraform import time_rotation.example 2020-02-12T06:36:13Z,2020-02-13T06:36:13Z
+-- exp-examples/resources/sleep/example_1.tf --
+# This resource will destroy (potentially immediately) after null_resource.next
+resource "null_resource" "previous" {}
+
+resource "time_sleep" "wait_30_seconds" {
+  depends_on = [null_resource.previous]
+
+  create_duration = "30s"
+}
+
+# This resource will create (at least) 30 seconds after null_resource.previous
+resource "null_resource" "next" {
+  depends_on = [time_sleep.wait_30_seconds]
+}
+-- exp-examples/resources/sleep/example_2.tf --
+# This resource will destroy (at least) 30 seconds after null_resource.next
+resource "null_resource" "previous" {}
+
+resource "time_sleep" "wait_30_seconds" {
+  depends_on = [null_resource.previous]
+
+  destroy_duration = "30s"
+}
+
+# This resource will create (potentially immediately) after null_resource.previous
+resource "null_resource" "next" {
+  depends_on = [time_sleep.wait_30_seconds]
+}
+-- exp-examples/resources/sleep/example_3.tf --
+resource "aws_ram_resource_association" "example" {
+  resource_arn       = aws_subnet.example.arn
+  resource_share_arn = aws_ram_resource_share.example.arn
+}
+
+# AWS resources shared via Resource Access Manager can take a few seconds to
+# propagate across AWS accounts after RAM returns a successful association.
+resource "time_sleep" "ram_resource_propagation" {
+  create_duration = "60s"
+
+  triggers = {
+    # This sets up a proper dependency on the RAM association
+    subnet_arn = aws_ram_resource_association.example.resource_arn
+    subnet_id  = aws_subnet.example.id
+  }
+}
+
+resource "aws_db_subnet_group" "example" {
+  name = "example"
+
+  # Read the Subnet identifier "through" the time_sleep resource to ensure a
+  # proper dependency and that both will change together.
+  subnet_ids = [time_sleep.ram_resource_propagation.triggers["subnet_id"]]
+}
+-- exp-examples/resources/sleep/import_1.sh --
+$ terraform import time_sleep.example 30s,
+-- exp-examples/resources/sleep/import_2.sh --
+$ terraform import time_sleep.example ,30s
+-- exp-examples/resources/static/example_1.tf --
+resource "time_static" "example" {}
+
+output "current_time" {
+  value = time_static.example.rfc3339
+}
+-- exp-examples/resources/static/example_2.tf --
+resource "time_static" "ami_update" {
+  triggers = {
+    # Save the time each switch of an AMI id
+    ami_id = data.aws_ami.example.id
+  }
+}
+
+resource "aws_instance" "server" {
+  # Read the AMI id "through" the time_static resource to ensure that
+  # both will change together.
+  ami = time_static.ami_update.triggers.ami_id
+
+  tags = {
+    AmiUpdateTime = time_static.ami_update.rfc3339
+  }
+
+  # ... (other aws_instance arguments) ...
+}
+-- exp-examples/resources/static/import_1.sh --
+$ terraform import time_static.example 2020-02-12T06:36:13Z
+-- exp-templates/index.md.tmpl --
+---
+layout: "time"
```

Comment:
If at all possible, we should skip the `layout` frontmatter. It is not necessary for provider documentation since it moved from terraform.io to the public registry and it can confuse provider developers. When we enable the checks command, this is likely something that should be flagged for removal (e.g. its already in `tfproviderdocs check`).